src/components/PublicityHeader/components/index.tsx

```diff
@@ -1,20 +1,24 @@
 import styles from './styles.module.scss';
 
-interface Props{
-    image:string;
-    text1:string;
-    text2:string;
+interface Props {
+    image: string;
+    text1: string;
+    text2?: string;
 }
-function InsideInfo({ image, text1, text2 } : Props) {
-  return (
-    <div className={styles.iconsSection}>
-      <img className={styles.icon} src={image} alt="" />
-      <div className={styles.text}>
-        <h2>{text1}</h2>
-        <p>{text2}</p>
-      </div>
-    </div>
-  );
+function InsideInfo({ image, text1, text2 }: Props) {
+    return (
+        <div className={styles.iconsSection}>
+            <img className={styles.icon} src={image} alt="" />
+            <div className={styles.text}>
+                <h2>{text1}</h2>
+                <p>{text2}</p>
+            </div>
+        </div>
+    );
 }
 
 export default InsideInfo;
+
+InsideInfo.defaultProps = {
+    text2: '',
```

Comment:
Since text2 is optional, it's ok to initialize it as undefined... also it helps with the comment above

Reply:
done